core/iwasm/aot/aot_loader.c

```diff
@@ -1392,6 +1400,13 @@ load_import_table_list(const uint8 **p_buf, const uint8 *buf_end,
             read_uint8(buf, buf_end, ref_type.ref_ht_common.nullable);
         }
 #endif
+        {
```

Comment:
should add `else` before `#endif`

Reply:
sync up with `aot_emit_table_info()`

```c
#if WASM_ENABLE_GC != 0
        if (comp_ctx->enable_gc
            && comp_data->import_tables[i].table_type.elem_ref_type) {
            EMIT_U8(comp_data->import_tables[i]
                        .table_type.elem_ref_type->ref_ht_common.nullable);
        }
        else
#endif
        {
            /* emit one placeholder to keep the same size */
            EMIT_U8(0);
        }

```

Reply:
ok, so you will add `else` before `#endif`?
```
#if WASM_ENABLE_GC != 0
        if (wasm_is_type_multi_byte_type(import_table->table_type.elem_type)) {
            read_uint8(buf, buf_end, ref_type.ref_ht_common.nullable);
        }
        else
#endif
        {
            ...
        }
```

Reply:
😨 oops